tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -413,6 +416,11 @@ public int PartialFlushMinSpans
         /// <seealso cref="ConfigurationKeys.KafkaCreateConsumerScopeEnabled"/>
         public bool KafkaCreateConsumerScopeEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to use the updated WCF instrumentation
+        /// </summary>
+        public bool WcfEnableNewInstrumentation { get; set; }
```

Comment:
Should we keep this internal if it's a feature flag that we might remove and enable by default?

Reply:
Yeah that's a good idea. Done in e75305ad01e62457aa7ce96009bbe6400439f0f5